src/cai_lab_to_nwb/zaki_2024/zaki_2024_metadata.yaml

```diff
@@ -1,44 +1,48 @@
 NWBFile:
   keywords:
     - hippocampus
-    - memory integration
+    - learning
     - memory-linking
     - ensemble
     - reactivation
-    - co-firing
+    - co-activity
     - offline periods
-    - sleep
-    - stress
-    - PTSD
+    - aversive
   related_publications:
-    - https://www.biorxiv.org/content/10.1101/2023.03.13.532469v2
+    - https://doi.org/10.1038/s41586-024-08168-4
   session_description:
-    A rich text description of the experiment. Can also just be the abstract of the publication.
+    In memory-linking behavioural experiments, mice were exposed to the neutral context for 10 min to explore. 
+    During aversive encoding, after a baseline period of 2 min, mice received three 2 s foot shocks of either amplitude 
+    0.25 mA (low-shock) or 1.5 mA (high-shock), with an intershock interval of 1 min. 
+    Then, 30 s after the final shock, the mice were removed and returned to the vivarium. 
+    On the next 3 days, the mice were tested in the previously experienced aversive and neutral contexts, 
+    as well as a completely novel context that they had not been exposed to previously, for 5 min each. 
+    The features of the neutral and novel contexts were counter-balanced and were made up of different olfactory, 
+    auditory, lighting and tactile cues. 
+    The aversive context was always the same with distinct cues from the neutral and novel contexts. 
+    In the low- versus high-shock experiments mice were tested in the aversive context first, 
+    followed by testing in the neutral and novel context counter-balanced; 
+    half of the mice received neutral recall and then novel-context exposure the next day, 
+    and the other half received novel-context exposure and then neutral recall.
+  experiment_description:
+    Simultaneous calcium imaging with Miniscopes and EEG/EMG experiment during retrospective memory-linking behavioral paradigm, 
+    to test whether ensemble co-reactivation is sleep-state specific
   institution: Icahn School of Medicine at Mount Sinai
   lab: Cai
   experimenter:
-    - Cai, Denise
```

Comment:
Mmm good point here, I have been following the convention that if the conversion is a paper I add all the people here but maybe that's better in dandi. I think the practical reason was that it would be difficult to confirm but maybe I am wrong. Did you confirm with him that he was the one doing all the experiments.

Reply:
Yes, I confirm with Joe that he was the experimenter for this dataset. (last email)

Reply:
OK.